### PR TITLE
Update Codecov package and add Codecov badge to README

### PR DESCRIPTION
**Pull request by thomasrockhu**
Tom from Codecov here. Looks like this repository wasn't sending coverage reports to Codecov for quite awhile. I updated the package to do so and added the Codecov badge.

#### Tests

- [x] The code changed/added as part of this PR has been covered with tests
- [x] All tests pass locally with `yarn test-unit`

#### Code Review

- [x] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

**Comment by TooTallNate** [comment]
So all you had to do was update the package?

**Comment by thomasrockhu** [comment]
@TooTallNate, seems like it. It could be a combination of fixes we made to the uploader and fixes we made recently. I would be inclined to believe it's the uploader, though.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 [![CI Status](https://badgen.net/github/checks/vercel/vercel?label=CI)](https://github.com/vercel/vercel/actions?workflow=CI)
 [![Join the community on GitHub Discussions](https://badgen.net/badge/join%20the%20discussion/on%20github/black?icon=github)](https://github.com/vercel/vercel/discussions)
+[![codecov](https://codecov.io/gh/vercel/vercel/branch/master/graph/badge.svg)](https://codecov.io/gh/vercel/vercel)
```
Comment (styfle) [inline README.md:11]: Let's not add a badge until we can get this number higher

**Review by styfle — COMMENTED** [review]
Thanks for the PR!

Does this take into account the monorepo paths?

I would expect `packages/now-cli/src/util/dev/router.ts` as the path, not `src/util/dev/router.ts` which is what its reporting.

Ideally, we would add code coverage to all the packages in this monorepo.

**Pull request closed without merging**